spec/rubocop/cop/rspec/factory_bot/dynamic_attribute_defined_statically_spec.rb

```diff
@@ -89,3 +89,4 @@
     end
   end
 end
+# rubocop:enable Metrics/LineLength
```

Comment:
you may re-enable right after the 4th line, which is the only one offending

Reply:
Fixed now @Darhazer